main.js

```diff
@@ -100,7 +100,17 @@ let feedback_screen = {
     preamble: FEEDBACK_PREAMBLE,
     questions: [
 	{prompt: FEEDBACK_PROMPT, rows: 5},
-    ]
+    ],
+    on_load: function() {
+        uil.saveJson(jsPsych.data.get().json(), ACCESS_KEY);
+    },
+    on_finish: function(data) {
+        let payload = {
+            feedback: data.response,
+            ...jsPsych.data.dataProperties // adds subject id and list info
```

Comment:
Like I said in the other comment. Perhaps the feedback should be anonymous, and this would couple the participant to his/her feedback.

Reply:
I think the idea was that people could also comment on things not working properly etc.
And even if we don't send the subject id, it's easy to derive it from timestamps